experiments/009_probablistic/readme.md

```diff
@@ -0,0 +1,90 @@
+is not necessarily fully calibrated. This is also done for non-night time (defined as true generation above 1% of the capacity), and
+for all times, including night time.
+
+Fraction of actual generation below the percentile for all times:
+
+| Horizon | 0.1   | 0.5      | 0.9      |
+|---------|-------|----------|----------|
+| 0       | 0.60806 | 0.77385  | 0.88295  |
+| 1       | 0.62665 | 0.78033  | 0.88061  |
+| 2       | 0.63707 | 0.78428  | 0.87586  |
+| 4       | 0.64015 | 0.78388  | 0.87372  |
+| 8       | 0.63601 | 0.77732  | 0.86664  |
+| 16      | 0.63801 | 0.77238    | 0.86396  |
+| 24      | 0.6328 | 0.77144    | 0.86811  |
+| 36      | 0.63687 | 0.77539   | 0.86797  |
+
+Fraction of actual generation below the percentile for only daylight (Generation > 0.01) times:
+
+| Horizon | 0.1   | 0.5   | 0.9   |
+|---------|-------|-------|-------|
+| 0       | 0.2478 | 0.56319| 0.769 |
+| 1       | 0.28589| 0.57784 | 0.7627 |
+| 2       | 0.30172 | 0.58307 | 0.75127 |
+| 4       | 0.30902 | 0.58279 | 0.74783 |
+| 8       | 0.30172 | 0.56999 | 0.73242 |
+| 16      | 0.30557 | 0.5586 | 0.72808 |
+| 24      | 0.29246 | 0.55904 | 0.73713 |
+| 36      | 0.29982 | 0.56576 | 0.73557 |
+
+## Results for cross-validation 2016-2022 mean range and quantile coverage
+
+| Horizon | 0.1     | 0.5     | 0.9     |
+|---------|---------|---------|---------|
+| 0       | 0.21619 | 0.49991 | 0.72409 |
+| 1       | 0.24395 | 0.49761 | 0.69554 |
+| 2       | 0.2563  | 0.488   | 0.66742 |
+| 3       | 0.25588 | 0.49109 | 0.66645 |
+| 4       | 0.25631 | 0.48897 | 0.66709 |
+| 5       | 0.25536 | 0.49024 | 0.66989 |
+| 6       | 0.25666 | 0.49079 | 0.669   |
+| 7       | 0.25537 | 0.48862 | 0.66705 |
+| 8       | 0.25444 | 0.48715 | 0.66836 |
+| 9       | 0.25524 | 0.48759 | 0.66886 |
+| 10      | 0.25443 | 0.48719 | 0.66661 |
+| 11      | 0.25302 | 0.48774 | 0.66849 |
+| 12      | 0.25294 | 0.4895  | 0.66953 |
+| 13      | 0.25381 | 0.48953 | 0.66902 |
+| 14      | 0.25379 | 0.48751 | 0.67067 |
+| 15      | 0.25463 | 0.49091 | 0.66932 |
+| 16      | 0.25517 | 0.48787 | 0.66843 |
+| 17      | 0.25394 | 0.48815 | 0.66976 |
+| 18      | 0.25636 | 0.48961 | 0.67008 |
+| 19      | 0.25522 | 0.48977 | 0.66885 |
+| 20      | 0.25505 | 0.48906 | 0.66857 |
+| 21      | 0.25536 | 0.49072 | 0.671   |
+| 22      | 0.25627 | 0.48986 | 0.66942 |
+| 23      | 0.25585 | 0.49085 | 0.67046 |
+| 24      | 0.25764 | 0.4914  | 0.67382 |
+| 25      | 0.25694 | 0.49036 | 0.66981 |
+| 26      | 0.25876 | 0.49247 | 0.67047 |
+| 27      | 0.26142 | 0.49424 | 0.67243 |
+| 28      | 0.25904 | 0.4941  | 0.6705  |
+| 29      | 0.26    | 0.49409 | 0.67048 |
+| 30      | 0.25909 | 0.49636 | 0.67121 |
+| 31      | 0.26019 | 0.49471 | 0.67071 |
+| 32      | 0.25914 | 0.49492 | 0.67213 |
+| 33      | 0.26286 | 0.49671 | 0.6728  |
+| 34      | 0.26168 | 0.49789 | 0.67299 |
+| 35      | 0.26133 | 0.48521 | 0.67172 |
+| 36      | 0.26343 | 0.49714 | 0.67256 |
```

Comment:
Could you add a comment about scaling the plevels and what values you have used?